scipy/stats/tests/test_mstats_basic.py

```diff
@@ -1085,6 +1085,24 @@ def test_zero_division(self):
             assert_array_equal(t, np.array([np.nan, np.nan]))
             assert_array_equal(p, np.array([np.nan, np.nan]))
 
+    def test_alternative(self):
```

Comment:
I think these `mstats_basic` tests are fine. Ideally, if this test is based on an existing two-sided test that compares `mstats` with `stats`, you could just parametrize that, but this is OK too.

Reply:
> Ideally, if this test is based on an existing two-sided test that compares `mstats` with `stats`, you could just parametrize that to go through all the alternatives, but this is OK too.

Makes sense. I will try to do that.